node/plain/functionApp/Tests/index.js

```diff
@@ -0,0 +1,7 @@
+var tests = require('require-dir')('./unittests');
+
+for (var functions in tests) {
```

Comment:
run it using gulp/grunt tasks - since they would publish the results to DevOps server

Reply:
Done, I've used gulp